rosdep/base.yaml

```diff
@@ -125,6 +125,11 @@ aravis-dev:
     '*': [libaravis-dev]
     bionic: null
     xenial: null
+aravis-tools:
+  debian: [aravis-tools]
+  ubuntu:
+    '*': [aravis-tools]
+    bionic: null
```

Comment:
@sgermanserrano hmm, I have the impression that `aravis-tools` and `aravis-tools-cli` packages in Debian can be found as a single `aravis` package in other distros (along with the rest). 

Would it make sense to collapse this entry into the existing `aravis` key and complete it? @clalancette for feedback.

Reply:
I think that modifying the already existing `aravis` rule so that it also includes ararvis-tools and aravis-tools-cli in all distros is fine.
We could also have four rules, `aravis`, `aravis-dev`, `aravis-tools-cli`, `aravis-tools`, but that seems like too many.

Reply:
sorted, I've collapse them into `aravis`